cc/sgx/test_operators/test_encdes.sh

```diff
@@ -0,0 +1,178 @@
+export LD_LIBRARY_PATH=.
```

Comment:
If we write UT, plz refer to her https://github.com/ieigen/ieigen/tree/main/cc/sgx/tests/unit_test.  or we can consider writing an example under `example` directory.